docs/contributors/e2e_tests.md

```diff
@@ -0,0 +1,57 @@
+# Execute end-to-end tests for local development
+
+To execute the end-to-end (e2e) test suite on your local development system:
+
+1. install `kind` (see https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
```

Comment:
Ideally, the codebase would specify _which_ version of kind and enforce that version is used in the tests.

Reply:
When I update the code in my next push this will be resolved since I have implemented `ctx` similar to what is done in OLM.